cli-artemis-jms/src/main/java/com/redhat/mqe/acc/Main.java

```diff
@@ -40,6 +40,9 @@
         @Binds
         ClientOptionManager bindClientOptionManager(AccClientOptionManager m);
 
+        @Binds
+        SenderClient bindSenderClient(AccSenderClient m);
+
```

Comment:
porad neverim ze tohle je potreba, ale u tebe se ukazalo ze je, a sam si stim ted hrat nebudu, takze ok

Reply:
Suhlasim, startClient() je celkom velky. Verim, ze transakcie budu dalsi split. A zrejme u viacerych klientov, nie len v acc. Ale to az casom. ;)